src/EnergyPlus/DataSizing.cc

```diff
@@ -637,26 +637,26 @@ Real64 calcDesignSpecificationOutdoorAir(EnergyPlusData &state,
 
     if (thisDSOA.numDSOA == 0) {
         // This is a simple DesignSpecification:OutdoorAir
-        return thisDSOA.calcOAFlowRate(state, ActualZoneNum, UseOccSchFlag, UseMinOASchFlag, PerPersonNotSet, MaxOAVolFlowFlag);
+        return thisDSOA.calcOAFlowRate(state, ActualZoneNum, UseOccSchFlag, UseMinOASchFlag, PerPersonNotSet, MaxOAVolFlowFlag, spaceNum);
     } else {
         // This is a DesignSpecification:OutdoorAir:SpaceList
         for (int dsoaCount = 1; dsoaCount <= thisDSOA.numDSOA; ++dsoaCount) {
-            totOAFlowRate += state.dataSize->OARequirements(thisDSOA.dsoaIndexes(dsoaCount))
-                                 .calcOAFlowRate(state,
-                                                 ActualZoneNum,
-                                                 UseOccSchFlag,
-                                                 UseMinOASchFlag,
-                                                 PerPersonNotSet,
-                                                 MaxOAVolFlowFlag,
-                                                 thisDSOA.dsoaSpaceIndexes(dsoaCount));
+            if ((spaceNum == 0) || ((spaceNum > 0) && (spaceNum == thisDSOA.dsoaSpaceIndexes(dsoaCount)))) {
```

Comment:
When calculating for a DesignSpecification:OutdoorAir:SpaceList, if `spaceNum==0` then sum all the spaces on the list, otherwise "sum" only the matching space. Same approach for 3 OA functions.

Reply:
Makes sense.